playwright/menu-availability.test.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from "@playwright/test";
+
+// Using logged in state from globalSteup
+test.use({ storageState: "playwright/artifacts/proStorageState.json" });
+
+test("test menu availability", async ({ page }) => {
+  // Try to go homepage
+  await page.goto("/");
```

Comment:
I we're going to test the menu itself, shouldn't we use clicks on the actual menu? Instead of going directly to the page. It kinda defeats the purpose of testing the menu.